prime-angular-frontend/src/app/modules/adjudication/shared/components/review-status-content/review-status-content.component.ts

```diff
@@ -33,6 +35,7 @@ class Reason {
     public documents?: BaseDocument[],
     public isSelfDeclaration?: boolean,
     public question?: string,
+    public potentialMatches?: number[],
```

Comment:
We'll want to rename this to indicate it is an array of IDs

Reply:
`reason.potentialMatchIds`

Reply:
renamed